Свидетельства о регистрации ПрЭВМ и БД

iseletkov opened the pull request:
Добавлено оформление свидетельств о государственной регистрации ПрЭВМ и БД в библиографии.

iseletkov committed:
Свидетельства о регистрации ПрЭВМ и БД

Добавлено оформление свидетельств о государственной регистрации ПрЭВМ и БД в библиографии.

iseletkov commented:
Прошу
1) проверить точность соответствия ГОСТу,
2) подсказать, как в choose проверять содержимое поля.

romanraspopov commented:
Прекрасно!
1. Кажется, все по ГОСТ.
2. Насколько я знаю, само значение поля проверить через choose - if нельзя. По крайней мере мне это в свое время так и не удалось. Проверяется только наличие какого-то значения в поле (т.е. пустое оно или нет). Проверка и выбор через choose - if работает только с нередактируемым полем type, которое действительно можно только выбирать (назначать) в Zotero.

romanraspopov commented:
Ваша проверка поля "note" (Extra или Дополнительно) прекрасно работает.

iseletkov committed:
Добавлен формат указа президента РФ.

Пример формата 
https://studresearsh.ru/4oformlenieIstochnikov/4_11_oformlenieUkazaPrezidenta.html
